reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,83 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.22.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.22.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+* Support for 64-bit only ARM systems for all supported Android versions.<!--AC-1890, AC-2428-->
+* Support for installing relevant 32-bit NVIDIA userspace binaries on AMD64 systems.<!--AC-2450-->
+* To allow easier identification and use of instances, you can now explicitly name an instance when creating it. See [How to create an instance](https://anbox-cloud.io/docs/howto/instance/create).<!--AC-2395-->
+* Support for Vulkan 1.3 for Android 13 images.<!--AC-2278-->
+* VHAL enhancements:
+    - Configuration returned by the APIs have a new field containing a name and a list of area names for the standard VHAL properties.<!--AC-2290 and AC-2289-->
+    - Property values returned by the APIs have a new field containing a list of area names for the standard VHAL properties.<!--AC-2290-->
+    - You can replace the Anbox Cloud VHAL implementation with your custom implementation for AAOS images. See [How to use a custom VHAL implementation](tbd). <!--AC-2371-->
+
+### Streaming stack improvements
+
+* Streaming can now be enabled when launching an instance using the `--enable-streaming` option. See [How to stream applications](https://anbox-cloud.io/docs/howto/application/stream) and [How to create an instance](https://anbox-cloud.io/docs/howto/instance/create).<!--Ac-2460 and AC-2360-->
+* VirGL is the default renderer for NVIDIA GPUs with driver version 545 and later. For driver versions earlier than 545, EmuGL is used as the default renderer. See [Rendering architecture](https://anbox-cloud.io/docs/explanation/rendering-architecture).<!--AC-2459, AC-2270 and AC-2348-->
```

Comment:
For driver versions earlier than 545, EmuGL is used as the default renderer.
->
 For driver versions earlier than 545 or old NVIDIA GPU architecture like Turing, Ampere or Ada, EmuGL is used as the default renderer.